package.json

```diff
@@ -19,9 +19,9 @@
     "hubot-xmpp"        : "git+https://github.com/markstory/hubot-xmpp.git#94c3438e42778c53e38f6909939c514da50a0dca",
     "hubot-hipchat"     : "git+https://github.com/StackStorm/hubot-hipchat#sonnyp-patch",
     "hubot-irc"         : "git+https://github.com/nandub/hubot-irc#51d1f4b418fc039a456b7891c7f9bf836699cfdc",
-    "hubot-slack"       : "^3.4.2"
+    "hubot-slack"       : "^4.3.4"
```

Comment:
From the reports in GitHub https://github.com/StackStorm/st2chatops/issues/56#issuecomment-285562378, https://github.com/StackStorm/st2docs/issues/418 and community, this should also fix proxy support for Slack adapter :+1: 
cc @ytjohn